connector/profiler/pom.xml

```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>me.bechberger</groupId>
       <artifactId>ap-loader-all</artifactId>
-      <version>3.0-8</version>
+      <version>3.0-9</version>
```

Comment:
Could you proceed this dependency update separately?

Reply:
Okay

Reply:
A separate PR:
https://github.com/apache/spark/pull/46427